2021-09-21 20:05:34,142 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 20:05:34,142 INFO    MainThread:32199 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200534-1hrd4ztd/logs/debug.log
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_200534-1hrd4ztd/logs/debug-internal.log
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():381] calling init triggers
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 20:05:34,143 INFO    MainThread:32199 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 2wg8eitb
2021-09-21 20:05:34,145 INFO    MainThread:32199 [wandb_run.py:finish():1275] finishing run mlwell-rgb/gta/2wg8eitb
2021-09-21 20:05:34,171 INFO    MainThread:32199 [jupyter.py:save_history():353] not saving jupyter history
2021-09-21 20:05:34,171 INFO    MainThread:32199 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 20:05:34,171 INFO    MainThread:32199 [wandb_init.py:_jupyter_teardown():281] cleaning up jupyter logic
2021-09-21 20:05:34,171 INFO    MainThread:32199 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-21 20:05:34,172 INFO    MainThread:32199 [wandb_run.py:_restore():1547] restore
2021-09-21 20:05:35,019 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 973
}

2021-09-21 20:05:35,472 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 973
}

2021-09-21 20:05:36,410 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 1300
}

2021-09-21 20:05:36,513 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 4838
}

2021-09-21 20:05:36,616 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 4838
}

2021-09-21 20:05:36,719 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 4838
}

2021-09-21 20:05:36,823 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 4838
}

2021-09-21 20:05:36,928 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 4838
}

2021-09-21 20:05:37,030 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 1326
  total_bytes: 4838
}

2021-09-21 20:05:37,133 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,238 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,341 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,444 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,548 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,650 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:37,846 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}

2021-09-21 20:05:38,876 INFO    MainThread:32199 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4838
  total_bytes: 4838
}
local_info {
}

2021-09-21 20:05:40,178 INFO    MainThread:32199 [wandb_run.py:_show_files():1947] logging synced files
2021-09-21 20:05:40,187 INFO    MainThread:32199 [wandb_init.py:init():430] starting backend
2021-09-21 20:05:40,187 INFO    MainThread:32199 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-21 20:05:40,191 INFO    MainThread:32199 [backend.py:ensure_launched():135] starting backend process...
2021-09-21 20:05:40,203 INFO    MainThread:32199 [backend.py:ensure_launched():139] started backend process with pid: 32308
2021-09-21 20:05:40,204 INFO    MainThread:32199 [wandb_init.py:init():435] backend started and connected
2021-09-21 20:05:40,231 INFO    MainThread:32199 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-21 20:05:40,231 INFO    MainThread:32199 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-21 20:05:40,231 INFO    MainThread:32199 [wandb_init.py:init():494] updated telemetry
2021-09-21 20:05:40,232 INFO    MainThread:32199 [wandb_init.py:init():517] communicating current version
2021-09-21 20:05:41,352 INFO    MainThread:32199 [wandb_init.py:init():522] got version response 
2021-09-21 20:05:41,352 INFO    MainThread:32199 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-21 20:05:42,407 INFO    MainThread:32199 [wandb_init.py:init():557] starting run threads in backend
2021-09-21 20:05:43,579 INFO    MainThread:32199 [wandb_run.py:_console_start():1605] atexit reg
2021-09-21 20:05:43,581 INFO    MainThread:32199 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-21 20:05:43,582 INFO    MainThread:32199 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-21 20:05:43,585 INFO    MainThread:32199 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-21 20:05:43,585 INFO    MainThread:32199 [wandb_init.py:init():582] run started, returning control to user process
2021-09-21 20:09:47,525 INFO    MainThread:32199 [wandb_run.py:finish():1275] finishing run mlwell-rgb/Set-Tree-graphtree/1hrd4ztd
